### DISCUSSION TITLE
Release Note Header Informations on mobile

### DISCUSSION THREAD

**Discussion by ciruz** [community/community#5995] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey,

the header informations (``__ days ago``, ``autor``, ``version``, ``commit``) for the new Release Notes looks like this on mobile:

![image](https://user-images.githubusercontent.com/987947/136021492-8bb963ce-8f18-4a85-baf1-e5f787eef6e5.png)

Maybe you can improve it slighty for mobile and seperate the informations a little bit more, something like this ( ``justify-content: space-between`` , ``flex: 1 0 50%;``, ``margin-right: 0``, etc. ):

![image](https://user-images.githubusercontent.com/987947/136022619-1fa15e3d-eda6-4630-adbf-415bf499cc1e.png)

**Comment by MylesBorins** [comment, marked as answer]
@ciruz thanks for bringing this one up as well! I noticed this too and haven't found time yet to document.

I'll add it to the list of UI tweaks for us to do on our next pass of design.

**Reply by MylesBorins** [reply]
as an update we are still planning to fix this and will begin work on it next week

**Reply by ciruz** [reply]
thank you for the update
